codewind-che-sidecar/tests/devfile.json

```diff
@@ -0,0 +1,65 @@
+{
+    "apiVersion": "1.0.0",
+    "metadata": {
+        "name": "codewind-che"
+    },
+    "projects": [
+        {
+            "name": "goproj",
+            "source": {
+                "type": "git",
+                "location": "https://github.com/microclimate-dev2ops/microclimateGoTemplate"
+            }
+        },
+        {
+            "name": "springproj",
+            "source": {
+                "type": "git",
+                "location": "https://github.com/maysunfaisal/maysunspring1.git"
+            }
+        },
+        {
+            "name": "swiftproj",
+            "source": {
+                "type": "git",
+                "location": "https://github.com/maysunfaisal/maysunswi.git"
+            }
+        },
+        {
+            "name": "microproj",
+            "source": {
+                "type": "git",
+                "location": "https://github.com/maysunfaisal/hellomicro.git"
+            }
+        },
+        {
+            "name": "nodeTemplate",
+            "source": {
+                "type": "git",
+                "location": "https://github.com/microclimate-dev2ops/nodeExpressTemplate"
+            }
+        }
+    ],
+    "components": [
+        {
+            "alias": "theia-ide",
+            "type": "cheEditor",
+            "id": "eclipse/che-theia/7.0.0-rc-3.0"
+        },
+        {
+            "alias": "exec-plugin",
+            "type": "chePlugin",
+            "id": "eclipse/che-machine-exec-plugin/0.0.1"
+        },
+	    {
+	        "alias": "codewind-sidecar",
+	        "type": "chePlugin",
+	        "id": "https://raw.githubusercontent.com/eclipse/codewind-che-plugin/master/plugins/codewind/codewind-sidecar/0.2.0/meta.yaml"
```

Comment:
We should change the version of the plugins used to `latest` instead of `0.2.0`.

Reply:
Cool thanks John! As we discussed earlier, I'll probably rework this bit to not use a static devfile we need to maintain separately, and just grab the live one from github and use a tool like `yq` or something to convert from yaml to json on the fly?